lib/pa11y-ci.js

```diff
@@ -67,18 +66,24 @@ function pa11yCi(urls, options) {
 
 		// This is the actual test runner, which the queue will
 		// execute on each of the URLs
-		function testRunner(config, done) {
-			const url = (typeof config === 'string' ? config : config.url);
+		async function testRunner(config) {
+			let url;
+			if (typeof config === 'string') {
+				url = config;
+				config = options;
+			} else {
+				url = config.url;
+				config = defaults({}, config, options);
+			}
 
 			// Run the Pa11y test on the current URL and add
 			// results to the report object
-			pa11yTest.run(config, (error, results) => {
-				if (error) {
-					log.error(` ${chalk.cyan('>')} ${url} - ${chalk.red('Failed to run')}`);
-					report.results[url] = [error];
-					return done();
-				}
-				const errors = results.filter(filterNonErrors);
+
+			try {
+				const results = await pa11y(url, config);
+
+				const errors = results.issues.filter(filterNonErrors);
```

Comment:
This filter shouldn't be needed any more, as Pa11y 5.x ignores warnings and notices by default, you can either just set `errors` to `results.issues` or maybe refactor to just reference `results.issues` everywhere instead of creating a new variable?